MANIFEST

```diff
@@ -48,3 +50,5 @@ t/data/fixtures/100-IseqRunStatus.yml
 t/data/fixtures/100-StudyUser.yml
 t/data/fixtures/200-IseqProductMetric.yml
 t/perlcriticrc
+META.yml
+META.json
```

Comment:
Meta.json and Meta.yml do not have to be in the MANIFEST file. They are generated when the package is built.